api/views/pois.py

```diff
@@ -59,35 +59,37 @@ def create_poi():
     data = request.get_json()
 
     # Check that all required parameters exist
-    if not all(i in data for i in ('name', 'description', 'map_year', 'x_coord', 'y_coord')):
+    required_fields = ['name', 'description', 'map_year', 'x_coord', 'y_coord']
+    missing_fields = [field for field in required_fields if data.get(field) is None]
+    if len(missing_fields):
         return create_response(
             status=422,
-            message='POI not created; missing required input parameter(s)'
+            message='POI not created; missing {}'.format(', '.join(missing_fields))
         )
-    data['links'] = data['links'] if 'links' in data else []
-    data['media'] = data['media'] if 'media' in data else []
-    data['story_ids'] = data['story_ids'] if 'story_ids' in data else []
+    content_fields = ['links', 'media', 'story_ids']
+    missing_content = [field for field in content_fields if type(data.get(field)) is not list]
+    for field in missing_content:
+        data[field] = []
     # Assumes that data['date'] is a parsable string containing a date if it exists

```

Comment:
can you add a newline above this (i like this [style rule](https://github.com/airbnb/javascript#comments--singleline))